modules/examples/eb-sf-batch/modulestack.yaml

```diff
@@ -0,0 +1,47 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This stack deploys a Module specific IAM permissions
+
+Parameters:
+  DeploymentName:
+    Type: String
+    Description: The name of the deployment
+  ModuleName:
+    Type: String
+    Description: The name of the Module
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+
+Resources:
+  Policy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Action:
+              - "s3:Create*"
+              - "s3:Put*"
+              - "s3:Delete*"
+              - "s3:Get*"
+              - "s3:List*"
+            Effect: Allow
+            Resource: ["arn:aws:s3:::addf*", "arn:aws:s3:::cdk*"]
```

Comment:
How does this limit work when bootstrapping with a different project name than addf?

Reply:
The example module here is to showcase the usage of the module with `addf` as the project name. customers can change it based on their naming pattern. also we encourage customers to name their projects something like `addf-clientX` so that `addf-*` will always help them